internal/resources/resource_lb_pools.go

```diff
@@ -64,30 +65,35 @@ func LoadBalancerPools() *schema.Resource {
 							Description:      "Network Loadbalancer Supported values are `LBSnatAutoMap`,`LBSnatDisabled`, `LBSnatIpPool`",
 						},
 						"passive_monitor_path": {
-							Type:        schema.TypeInt,
-							Optional:    true,
-							Description: "passive_monitor_path for Network loadbalancer pool",
+							Type:     schema.TypeInt,
+							Optional: true,
+							Description: "Passive Monitor ID, Get the `Id` from " + DSLBMonitor +
+								"datasource to obtain the passive monitor ID",
 						},
 						"active_monitor_paths": {
-							Type:        schema.TypeInt,
-							Optional:    true,
-							Description: "active_monitor_paths for Network loadbalancer pool",
+							Type:     schema.TypeInt,
+							Optional: true,
+							Description: "Active Monitor ID, Get the `Id` from " + DSLBMonitor +
+								"datasource to obtain the active monitor ID",
```

Comment:
this sentence looks proper for the a use case of Data Source mentioning. small comment is it `Id` and `id` ?

Reply:
updated